plugins/inputs/win_perf_counters/win_perf_counters.go

```diff
@@ -367,7 +367,7 @@ func (m *WinPerfCounters) ParseConfig() error {
 					err := m.AddItem(counterPath, computer, objectname, instance, counter, PerfObject.Measurement, PerfObject.IncludeTotal, PerfObject.UseRawValues)
 					if err != nil {
 						if PerfObject.FailOnMissing || PerfObject.WarnOnMissing {
-							m.Log.Errorf("invalid counterPath: '%s'. Error: %s\n", counterPath, err.Error())
+							m.Log.Errorf("invalid counterPath %q: %w", counterPath, err)
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.